.github/workflows/pb-update-go.yml

```
@@ -1,7 +1,7 @@
 name: Update Go
 "on":
     schedule:
-        - cron: 9 2 * * 1
+        - cron: 27 2 * * 1
```

```suggestion
        - cron: 9 2 * * 1
```

Don't change this. When you run pipeline-builder, it will sometimes want to change the randomized minute when the update runs. I thought I fixed this, but not sure. Anyway, just avoid committing a change to the cron scheduled minute.